### PR TITLE
File upload

### PR DESCRIPTION
**Pull request by sei-byoung**
Implement file upload functionality

### CODE REVIEW

**Review by sei-tspencer — APPROVED** [review]

**Review by sei-tspencer — APPROVED** [review]

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -35,6 +37,17 @@ export const ROUTES: Routes = [
     component: AdminAppComponent,
     canActivate: [ComnAuthGuardService],
   },
+  // Placeholder while developing file browsing component
```
Comment (sei-aschlackman) [inline src/app/app-routing.module.ts:40]: is this comment still valid or can it be removed?

File: src/app/components/admin-app/admin-view-search/admin-view-edit/admin-view-edit.component.scss [hunk 2]
```diff
@@ -45,3 +45,24 @@
...
+  flex-direction: row;
+  justify-content: left;
+  align-items: center;
+}
+
+#teamSelect {
+  margin-right: 25px;
+}
+
+// .btnList {
```
Comment (sei-aschlackman) [inline src/app/components/admin-app/admin-view-search/admin-view-edit/admin-view-edit.component.scss:60]: can this be removed?

**Review by sei-ebram — APPROVED** [review]

File: src/app/components/player/file-browse/file-browse.component.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    private fileService: FileService,
+    private teamService: TeamService,
+    private route: ActivatedRoute, 
+  ) { }
+
+  ngOnInit(): void {
+    this.getFiles();
+  }
+
+  getFiles(): void {
```
Comment (sei-noconnor) [inline src/app/components/player/file-browse/file-browse.component.ts:38]: You have nested subscribes here, look into switchMap, mergeMap if you have dependant child observables, or combineLatest and withLatestFrom if you need to combine multiple observable streams.

File: src/app/components/player/open-file/open-file.component.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+})
+export class OpenFileComponent implements OnInit {
+
+  constructor(
+    private fileService: FileService,
+    private route: ActivatedRoute,
+  ) { }
+
+  ngOnInit(): void {
+    this.route.queryParams.subscribe(params => {
```
Comment (sei-noconnor) [inline src/app/components/player/open-file/open-file.component.ts:28]: More nested subscriptions, see previous comment.

**Review by sei-noconnor — APPROVED** [review]

**Pull request merged**